### PR TITLE
Metafiles

### PR DESCRIPTION
**Pull request by quincylin1**
added metafiles for mmocr

### CODE REVIEW

File: configs/kie/sdmgr/metafile.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+Collections:
+- Name: SDMGR
+  Metadata:
+    Training Data: KIEDataset
+    Training Techniques:
+      - Adam
+    Training Resources: 4x NVIDIA V100 GPUs
```
Comment (cuhk-hbsun) [inline configs/kie/sdmgr/metafile.yml:7]:
```suggestion
    Training Resources: 1x NVIDIA 1080Ti GPU
```

File: configs/kie/sdmgr/metafile.yml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    Architecture:
+      - UNet
+  Paper: https://arxiv.org/abs/2103.14470.pdf
+  README: configs/kie/sdmgr/README.md
+
+Models:
+  - Name: sdmgr_novisual_60e_wildreceipt
+    In Collection: SDMGR
+    Config: configs/kie/sdmgr/sdmgr_novisual_60e_wildreceipt.py
```
Comment (cuhk-hbsun) [inline configs/kie/sdmgr/metafile.yml:16]:
```suggestion
    Config: configs/kie/sdmgr/sdmgr_unet16_60e_wildreceipt.py
```

File: configs/kie/sdmgr/metafile.yml [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    Results:
+      - Task: Key Information Extraction
+        Dataset: wildreceipt
+        Metrics:
+          macro_f1: 0.876
+    Weights: https://download.openmmlab.com/mmocr/kie/sdmgr/sdmgr_unet16_60e_wildreceipt_20210405-16a47642.pth
+
+  - Name: sdmgr_unet16_60e_wildreceipt
+    In Collection: SDMGR
+    Config: configs/kie/sdmgr/sdmgr_unet16_60e_wildreceipt.py
```
Comment (cuhk-hbsun) [inline configs/kie/sdmgr/metafile.yml:28]:
```suggestion
    Config: configs/kie/sdmgr/sdmgr_novisual_60e_wildreceipt.py
```

File: configs/textdet/dbnet/metafile.yml [hunk 4]
```diff
@@ -0,0 +1,37 @@
+Collections:
+- Name: DBNet
+  Metadata:
+    Training Data: ICDAR2015
+    Training Techniques:
+      - SGD with Momentum
+    Training Resources: 8x NVIDIA V100 GPUs
```
Comment (cuhk-hbsun) [inline configs/textdet/dbnet/metafile.yml:7]:
```suggestion
    Training Resources: 8x NVIDIA 1080Ti GPUs
```

File: configs/textdet/maskrcnn/metafile.yml [hunk 5]
```diff
@@ -0,0 +1,49 @@
+Collections:
+- Name: Mask R-CNN
+  Metadata:
+    Training Data: ICDARDataset
+    Training Techniques:
+      - SGD with Momentum
+    Training Resources: 8x NVIDIA V100 GPUs
```
Comment (cuhk-hbsun) [inline configs/textdet/maskrcnn/metafile.yml:7]:
```suggestion
    Training Resources: 8x NVIDIA 1080Ti GPUs
```

File: configs/textdet/panet/metafile.yml [hunk 6]
```diff
@@ -0,0 +1,49 @@
+Collections:
+- Name: PANet
+  Metadata:
+    Training Data: ICDARDataset
+    Training Techniques:
+      - Adam
+    Training Resources: 8x NVIDIA V100 GPUs
```
Comment (cuhk-hbsun) [inline configs/textdet/panet/metafile.yml:7]:
```suggestion
    Training Resources: 8x NVIDIA 1080Ti GPUs
```

File: configs/textdet/psenet/metafile.yml [hunk 7]
```diff
@@ -0,0 +1,49 @@
+Collections:
+- Name: PSENet
+  Metadata:
+    Training Data: ICDARDataset
+    Training Techniques:
+      - Adam
+    Training Resources: 8x NVIDIA V100 GPUs
```
Comment (cuhk-hbsun) [inline configs/textdet/psenet/metafile.yml:7]:
```suggestion
    Training Resources: 8x NVIDIA 1080Ti GPUs
```

File: configs/textdet/psenet/metafile.yml [hunk 8]
```diff
@@ -0,0 +1,49 @@
...
+    Results:
+      - Task: Instance Segmentation
+        Dataset: ICDAR2015
+        Metrics:
+          hmean-iou: 0.807
+    Weights: https://download.openmmlab.com/mmocr/textdet/psenet/psenet_r50_fpnf_600e_icdar2015-c6131f0d.pth
+
+  - Name: psenet_r50_fpnf_600e_icdar2017
+    In Collection: PSENet
+    Config: configs/textdet/psenet/psenet_r50_fpnf_600e_icdar2017.py
```
Comment (cuhk-hbsun) [inline configs/textdet/psenet/metafile.yml:41]:
```suggestion
    configs/textdet/psenet/psenet_r50_fpnf_600e_icdar2015.py
```

File: configs/textdet/psenet/metafile.yml [hunk 9]
```diff
@@ -0,0 +1,49 @@
...
+    Metadata:
+      Training Data: ICDAR2015
+    Results:
+      - Task: Instance Segmentation
+        Dataset: ICDAR2015
+        Metrics:
+          hmean-iou: 0.807
+    Weights: https://download.openmmlab.com/mmocr/textdet/psenet/psenet_r50_fpnf_600e_icdar2015-c6131f0d.pth
+
+  - Name: psenet_r50_fpnf_600e_icdar2017
```
Comment (cuhk-hbsun) [inline configs/textdet/psenet/metafile.yml:39]:
```suggestion
  - Name: psenet_r50_fpnf_600e_icdar2015_with_pretrain
```

File: configs/textrecog/crnn/metafile.yml [hunk 10]
```diff
@@ -0,0 +1,31 @@
+Collections:
+- Name: CRNN
+  Metadata:
+    Training Data: OCRDataset
+    Training Techniques:
+      - Adadelta
+    Training Resources: 8x NVIDIA V100 GPUs
```
Comment (cuhk-hbsun) [inline configs/textrecog/crnn/metafile.yml:7]:
```suggestion
    Training Resources: 4x NVIDIA 1080Ti GPUs
```

File: configs/textrecog/nrtr/metafile.yml [hunk 11]
```diff
@@ -0,0 +1,74 @@
+Collections:
+- Name: NRTR
+  Metadata:
+    Training Data: OCRDataset
+    Training Techniques:
+      - Adam
+    Training Resources: 8x NVIDIA V100 GPUs
```
Comment (cuhk-hbsun) [inline configs/textrecog/nrtr/metafile.yml:7]:
```suggestion
    Training Resources: 64x NVIDIA 1080Ti GPUs
```

File: configs/textrecog/nrtr/metafile.yml [hunk 12]
```diff
@@ -0,0 +1,74 @@
...
+      - ResNet31OCR
+      - TFEncoder
+      - TFDecoder
+  Paper: https://arxiv.org/pdf/1806.00926.pdf
+  README: configs/textrecog/nrtr/README.md
+
+Models:
+  - Name: nrtr_r31_1by8_1by4_academic
+    In Collection: NRTR
+    Config: configs/textrecog/nrtr/nrtr_r31_1by8_1by4_academic.py
```
Comment (cuhk-hbsun) [inline configs/textrecog/nrtr/metafile.yml:18]:
```suggestion
    Config: configs/textrecog/nrtr/nrtr_r31_1by16_1by8_academic.py
```

File: configs/textrecog/nrtr/metafile.yml [hunk 13]
```diff
@@ -0,0 +1,74 @@
...
+        Metrics:
+          acc:
+            - 74.5
+            - 78.5
+            - 86.5
+    Weights: https://download.openmmlab.com/mmocr/textrecog/nrtr/nrtr_r31_academic_20210406-954db95e.pth
+
+  - Name: nrtr_r31_1by16_1by8_academic
+    In Collection: NRTR
+    Config: configs/textrecog/nrtr/nrtr_r31_1by16_1by8_academic.py
```
Comment (cuhk-hbsun) [inline configs/textrecog/nrtr/metafile.yml:48]:
```suggestion
    Config: configs/textrecog/nrtr/nrtr_r31_1by8_1by4_academic.py
```

File: configs/textrecog/nrtr/metafile.yml [hunk 14]
```diff
@@ -0,0 +1,74 @@
...
+          - SVTP
+          - CT80
+        Metrics:
+          acc:
+            - 74.5
+            - 78.5
+            - 86.5
+    Weights: https://download.openmmlab.com/mmocr/textrecog/nrtr/nrtr_r31_academic_20210406-954db95e.pth
+
+  - Name: nrtr_r31_1by16_1by8_academic
```
Comment (cuhk-hbsun) [inline configs/textrecog/nrtr/metafile.yml:46]:
```suggestion
  - Name: nrtr_r31_1by8_1by4_academic
```

File: configs/textrecog/nrtr/metafile.yml [hunk 15]
```diff
@@ -0,0 +1,74 @@
...
+    Training Resources: 8x NVIDIA V100 GPUs
+    Architecture:
+      - ResNet31OCR
+      - TFEncoder
+      - TFDecoder
+  Paper: https://arxiv.org/pdf/1806.00926.pdf
+  README: configs/textrecog/nrtr/README.md
+
+Models:
+  - Name: nrtr_r31_1by8_1by4_academic
```
Comment (cuhk-hbsun) [inline configs/textrecog/nrtr/metafile.yml:16]:
```suggestion
  - Name: nrtr_r31_1by16_1by8_academic
```

File: configs/textrecog/robust_scanner/metafile.yml [hunk 16]
```diff
@@ -0,0 +1,50 @@
+Collections:
+- Name: RobustScanner
+  Metadata:
+    Training Data: OCRDataset
+    Training Techniques:
+      - Adam
+    Training Resources: 8x NVIDIA V100 GPUs
```
Comment (cuhk-hbsun) [inline configs/textrecog/robust_scanner/metafile.yml:7]:
```suggestion
    Training Resources: 16x NVIDIA 1080Ti GPUs
```

File: configs/textrecog/sar/metafile.yml [hunk 17]
```diff
@@ -0,0 +1,98 @@
+Collections:
+- Name: SAR
+  Metadata:
+    Training Data: OCRDataset
+    Training Techniques:
+      - Adam
+    Training Resources: 48x NVIDIA V100 GPUs
```
Comment (cuhk-hbsun) [inline configs/textrecog/sar/metafile.yml:7]:
```suggestion
    Training Resources: 48x NVIDIA 1080Ti GPUs
```

File: configs/textrecog/seg/metafile.yml [hunk 18]
```diff
@@ -0,0 +1,35 @@
+Collections:
+- Name: SEG
+  Metadata:
+    Training Data: mixture
+    Training Techniques:
+      - Adam
+    Training Resources: 8x NVIDIA V100 GPUs
```
Comment (cuhk-hbsun) [inline configs/textrecog/seg/metafile.yml:7]:
```suggestion
    Training Resources: 4x NVIDIA 1080Ti GPUs
```

File: configs/textrecog/tps/metafile.yml [hunk 19]
```diff
@@ -0,0 +1,30 @@
...
+    Metadata:
+      Training Data: Syn90k
+    Results:
+      - Task: Regular Text Recognition
+        Dataset:
+        - IIIT5K
+        - SVT
+        - ICDAR2013
+        Metrics:
+          acc:
```
Comment (cuhk-hbsun) [inline configs/textrecog/tps/metafile.yml:29]: update this with the latest readme in main branch

**Comment by innerlee** [comment]
The diff looks wrong. You may check if there are errors during rebasing

**Pull request closed without merging**